### PR TITLE
[SPARK-45343][DOCS] Clarify behavior of multiLine in CSV options

### PR DESCRIPTION
**Pull request by wrschneider**
### What changes were proposed in this pull request?

this is a documentation-only change to clarify CSV `multiLine` option:
https://issues.apache.org/jira/browse/SPARK-45343

### Why are the changes needed?

documentation clarity

### Does this PR introduce _any_ user-facing change?

Documentation only

### How was this patch tested?

N/A, documentation only

### Was this patch authored or co-authored using generative AI tooling?

Documentation only

### CODE REVIEW

File: docs/sql-data-sources-csv.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ Data source options of CSV can be set via:
   <tr>
     <td><code>multiLine</code></td>
     <td>false</td>
-    <td>Parse one record, which may span multiple lines, per file. CSV built-in functions ignore this option.</td>
+    <td>Allows a row to span multiple lines, by parsing line breaks within quoted values as part of the value itself. CSV built-in functions ignore this option.</td>
```
Comment (srowen) [inline docs/sql-data-sources-csv.md:216]: What's the 'fix' here, or is it just rewording?

Comment (wrschneider) [reply docs/sql-data-sources-csv.md:216]: "Parse one record... *per file*" sounds like the whole file is going to be parsed as a single record.

**Comment by srowen** [comment]
It looks like tests aren't enabled in your branch?

**Comment by wrschneider** [comment]
> It looks like tests aren't enabled in your branch?

Fixed, checks pass now

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**